vagrant/install.sh

```diff
@@ -24,6 +24,9 @@ sudo apt-get -qq update
 echo "Installing tagging tools & other dependencies..."
 sudo apt-get -qq install -y AtomicParsley flac vorbis-tools imagemagick oracle-java8-installer elasticsearch pkg-config yasm libfaac-dev libmp3lame-dev libvorbis-dev libtheora-dev
 
+echo "Installing PHP extensions"
+sudo apt-get -qq install -y libgmp-dev php-gmp
```

Comment:
These should be combined into the `apt-get` command above - it's faster to do as much as we can with `apt-get` at once.